scripts/run_validator_node.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+
+set -ue
+set -o pipefail
+
+sudo_cmd=""
+files_path=""
+chain_spec_name=""
+keychain_exists=""
+session_keys=""
+binary_path=""
+
+echo "This script will setup a Atleta Validator on your PC. Press Ctrl-C at any time to cancel."
+echo "Checking privileges... "
+if [ $(id -u) -eq 0 ]; then
+    echo "OK (root)"
+elif command -v sudo &> /dev/null; then
+    echo "OK (sudo)"
+    sudo_cmd=sudo
+else
+    echo "FAIL"
+    echo "Not running as root and no sudo detected. Please run this as root or configure sudo. Exiting." >&2
+    exit 1
+fi
+
+echo "Detecting your architecture... "
+operating_system=$(uname -s)
+if [ "$operating_system" != "Linux" -a "$operating_system" != "Darwin" ]; then
+    echo "$operating_system is currently not supported by Atleta Node, only Linux or Darwin are supported. Exiting." >&2
+    exit 1
+else
+    echo "OK ($operating_system)"
+fi
+
+if [[ $operating_system == "Linux" ]]; then
+    echo -n "Checking for systemd... "
+    if [ -e /run/systemd/system ]; then
+        echo "OK"
+    else
+        echo "FAIL"
+        echo "No systemd detected. Exiting." >&2
+        exit 1
+    fi
+fi
+
+if ! command -v jq &>/dev/null || \
+    ! command -v curl &>/dev/null
+then
+    echo "You need to install some dependencies before continuing:"
+    echo "  jq curl"
+    exit 1
+fi
+
+while [ "$chain_spec_name" == "" ]; do
+    echo "Available networks: "
+    echo "1) Devnet"
+    echo "2) Testnet"
+    echo "3) Mainnet"
+    read -p "Select network for your running node: " choice
+    case $choice in
+        1)
+            chain_spec_name="devnet"
+            ;;
+        2)
+            chain_spec_name="testnet"
+            ;;
+        3)
+            chain_spec_name="mainnet"
+            ;;
+        *)
+            echo "Invalid choice. Please select a right number."
+            continue
+            ;;
+    esac
+done
+
+echo "Your choice is: $chain_spec_name network."
+
+while [ -z "$files_path" ]; do
+    read -p "Enter the path where you want to store at least a few gigabytes of data (or press Enter to use the standard $HOME/atleta/chain directory): " files_path
+
+    if [ -z "$files_path" ]; then
+        files_path="$HOME/atleta/chain"
+        binary_path="/opt/atleta"
```

Comment:
Why binary path differs and depends on what _files_ path we selected?

Reply:
check operation system files